nocts_cata_mod_BN/Surv_help/c_armor.json

```
@@ -620,7 +620,7 @@
     "description": "A simple leather bikini bottom.",
     "material": [ "leather" ],
     "warmth": 5,
-    "delete": { "flags": [ "WATER_FRIENDLY" ] }
+    "extend": { "flags": [ "FANCY", "WATER_FRIENDLY" ] }
```

```suggestion
    "delete": { "flags": [ "WATER_FRIENDLY" ] },
    "extend": { "flags": [ "FANCY" ] }
```
Leather and fur bikini items in vanilla lose the `WATER_FRIENDLY` flag for whatever reason, which is why they're deleted here.